include/deal.II/base/tensor_deprecated.h

```diff
@@ -96,6 +123,31 @@ Number determinant (const Tensor<1,1,Number> &t) DEAL_II_DEPRECATED;
 
 #ifndef DOXYGEN
```

Comment:
Out of curiosity, why do you do this? We now let doxygen parse all of the .cc files as well, so that it can put references to the implementation into the documentation.